Sharlayan/Models/Signature.cs

```diff
@@ -87,9 +91,15 @@ public IntPtr GetAddress()
             {
                 return baseAddress;
             }
-            return MemoryHandler.Instance.ResolvePointerPath(PointerPath, baseAddress, IsASMSignature);
+            if (_resolvedPointerPath == IntPtr.Zero)
+            {
+                _resolvedPointerPath = MemoryHandler.Instance.ResolvePointerPath(PointerPath, baseAddress, IsASMSignature);
```

Comment:
Is it safe to cache the resolved pointer path? What if the memory address is reallocated? for example, when zoning, addresses?  Not sure how you handle Chatlog pointers but they do change when the buffer becomes full.

Reply:
Um... good question. For chatlog pointers (at least offsets) I ensure indexes on each scan of the chatlog array. But the initial offset I deal with shouldn't change on zoning though??? At least it never has for me.

Did you find it changes?

Reply:
I didn't test.  probably just roll this one change back since the rest is good?